.github/workflows/reusable-test.yml

```diff
@@ -119,6 +119,14 @@ jobs:
         mkdir ${{github.workspace}}\${{env.BUILD_PLATFORM}}\${{env.BUILD_CONFIGURATION}}\TestLogs
         wpr.exe -start ${{github.workspace}}\${{env.BUILD_PLATFORM}}\${{env.BUILD_CONFIGURATION}}\ebpfforwindows.wprp -filemode
 
+    - name: Configure eBPF store
```

Comment:
is this needed for all tests? If not put a "if:" condition maybe? If cmd shell is not needed you can skip shell: cmd and use ".\" before exe name to run them.

Reply:
Ideally this is needed for all tests except the driver tests, so I added it for all tests.